docs/detections/detections-exclude-cold-frozen-data-tiers.asciidoc

```diff
@@ -1,18 +1,24 @@
 [[exclude-cold-frozen-data-individual-rules]]
-== Exclude cold and frozen data from a rule
+== Exclude cold and frozen data from rule executions
 
 :frontmatter-description: Configure a rule to ignore cold and frozen data during execution. 
 :frontmatter-tags-products: [security]
 :frontmatter-tags-content-type: [how-to]
 :frontmatter-tags-user-goals: [manage]
 
-Rules that query cold and frozen data might perform more slowly. To exclude cold and frozen data, add a Query DSL filter that ignores cold and frozen {ref}/data-tiers.html[data tiers] when executing. You can add the filter when creating a new rule or updating an existing one. 
+Rules that query cold and frozen {ref}/data-tiers.html[data tiers] might perform more slowly. To exclude query results from cold and frozen tiers, add a Query DSL filter that ignores cold and frozen documents when executing. This can help Elasticsearch exclude cold and frozen data more efficiently. You can add the filter when creating a new rule or updating an existing one.
 
-NOTE: This method is not supported for {esql} and {ml} rules.
+TIP: To ensure that _all_ rules in a {kib} space exclude cold and frozen documents when executing, configure the `excludedDataTiersForRuleExecution` <<exclude-cold-frozen-data-rule-executions,advanced setting>>.
 
-TIP: To ensure that _all_ rules in a {kib} space exclude cold and frozen data when executing, configure the `excludedDataTiersForRuleExecution` <<exclude-cold-frozen-data-rule-executions,advanced setting>>.
+[IMPORTANT]
+====
 
-Here is a sample Query DSL filter that excludes frozen tier data from a rule's execution:
+* This method is not supported for {esql} and {ml} rules.
```

Comment:
The setting is supported for ES|QL rules and may help performance when frozen shards are available, but it won't prevent failures when frozen shards are unavailable. The setting has no effect on ML rules.

Reply:
> The setting is supported for ES|QL rules and may help performance when frozen shards are available,

@marshallmain to clarify, are you saying that using a **DSL filter** to exclude query results from cold and frozen tiers does work for ES|QL rules? I know the filter works similarly to the `excludedDataTiersForRuleExecution` setting, but I still want to clarify in case my understanding isn't correct.

Reply:
Sorry I missed that this was about adding the filter manually rather than the advanced setting - it's correct as is 👍